ami/instance.tf

```diff
@@ -26,10 +25,14 @@ resource "aws_instance" "ubuntu-ami-template-instance" {
   # Our instance-user-data.sh script is built by Terraform at
   # apply-time so that it can put additional files onto the
   # instance. For more information see the definition of this resource.
-  user_data = data.template_file.ubuntu_instance_user_data.rendered
+  user_data = templatefile("ubuntu-instance-user-data.tpl.sh", {
+      docker_apt_key = data.local_file.docker_apt_key.content
+    }
+  )
 
-  subnet_id = data.aws_subnet.ccdl_dev_subnet.id
   associate_public_ip_address = true
+  key_name = "data-refinery-key-circleci-prod"
```

Comment:
```suggestion
  key_name = "data-refinery-key-${var.user}-${var.stage}"
```

Reply:
Use templated values so there isn't collision for dev/stage/prod

Reply:
Not sure if these variables are applicable in `ami` context. Keeping as is for now.